operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -115,6 +115,15 @@
           + "Defaults to true if `domainHomeSourceType` is PersistentVolume; false, otherwise.")
   private Boolean logHomeEnabled; // Boolean object, null if unspecified
 
+  /**
+   * Whether the port forwarding is enabled.
+   */
+  @Description(
+          "Specifies whether the port forwarding is enabled. "
```

Comment:
Most customers won't understand what this means. Perhaps something like, "When this flag is enabled, the operator enhances the domain configuration with a NetworkAccessPoint for the admin protocol that listens on localhost. This allows access to the Administration Console using the `kubectl port-forward` pattern, which is often more secure than exposing the console to the public Internet. Defaults to true."

Reply:
Fixed.